discord/ext/commands/bot.py

```diff
@@ -816,7 +863,7 @@ def _resolve_name(self, name: str, package: Optional[str]) -> str:
         except ImportError:
             raise errors.ExtensionNotFound(name)
 
-    async def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
+    async def load_extension(self, name: str, /, *, package: Optional[str] = None) -> None:
```

Comment:
I think this one is fine as a dual keyword/positional.

Reply:
```suggestion
    async def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
```